torchmetrics/image/ssim.py

```diff
@@ -23,20 +23,24 @@
 
 
 class StructuralSimilarityIndexMeasure(Metric):
-    """Computes Structual Similarity Index Measure (SSIM_).
+    """Computes Structual Similarity Index Measure.
```

Comment:
what happened to the link?
```suggestion
    """Computes Structual Similarity Index Measure (SSIM_).
```